CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ Yii Framework 2 apidoc extension Change Log
 -----------------------
 
 - Enh #185: Use HTTPS for www.php.net links (kamarton)
+- Enh #137: @since tags are now propagated to inherited methods/properties and magic properties (brandonkelly)
```

Comment:
Are you sure about that? This will create invalid annotations in some cases. Especially for methods/properties inherited from class from different package (which may have different version than package which extends this class).

Reply:
Yeah good point. Will pick up that conversation over on #137. Reverted that change from this PR in the meantime.

Reply:
@rob006 Just submitted https://github.com/yiisoft/yii2-apidoc/pull/190 which is a second attempt at this, but this time factors in the method/property's package name, and only copies `@since` over if the package names match.